automation/src/test/scala/org/broadinstitute/dsde/workbench/WebBrowserSpec.scala

```diff
@@ -1,22 +1,19 @@
-package org.broadinstitute.dsde.firecloud.test
```

Comment:
File summary: remove some Firecloud-specific code, and add download code from a UI Swatomation branch